### DISCUSSION TITLE
Preview Deployment for "Docker Image" Resouces

### DISCUSSION THREAD

**Discussion by cngJo** [coollabsio/coolify#4025] [Feature Requests]
I'm currently switching to pushing my images to a custom registry (`registry.example.tld`) instead of letting the Server build the images. 

I'm pushing docker images from GHA and hitting the API to deploy the image with curl. 

It would be awesome to support preview deployment with the API by passing a PR number and docker tag to it 

sth. like:

```bash 
/api/v1/deploy-preview?uuid=<uuid>&docker_tag=pr_1234&pr_id=1234"
```

I'd be willing to give it a shot implementing myself, if you'd accept such a PR ;)

**Comment by peaklabs-dev** [comment]
Of course, PRs are always welcome.

**Comment by andrasbacsai** [comment]
I will add this to the next version v471
